### PR TITLE
(MODE-10433) Application documentation

### PR DESCRIPTION
**Pull request by scmono**
HAL documentation included

### CODE REVIEW

**Comment by scmono** [comment]
I've added a bit for each app. Billing could use a bit of fleshing out as I wasn't sure about that one.

File: presentations/02_apps/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ - SAT 
+ - Reseller Config
+ - Billing
+
+---
+
+# HAL
+
+The main backbone of the Mono administration.
+This is where all products are managed, you can start sites, it handles authtentication of the login requests.
```
Comment (siozosdr) [inline presentations/02_apps/README.md]: authentication

File: presentations/02_apps/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+We are currently slowly in the process of seperating some of the concerns away from HAL, so it doesn't have to know the specifics for each registry etc.
```
Comment (siozosdr) [inline presentations/02_apps/README.md]: separating

File: presentations/02_apps/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+SAL is primarily a cronjob extracting data from Google Analytics®, where we then process it into the daily states that we return.
+
+Sadly this means that we're always a day behind on interactions with the site.
+
+
+---
+
+# RAI
+When a reseller doesn't want to use the HAL api directly, they often have access to RAI.
+Our own support also uses RAI to asses most of the support issues they get.
```
Comment (siozosdr) [inline presentations/02_apps/README.md]: assess

File: presentations/03_HAL/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+ - Domain Handling
+ - Subscriptions
+ 
+---
+
+# Function Overview
+
+HAL is our central repository for subscription handling, billing, centralised data processing and reporting. It runs primarily on util01-cph.mono.
+
+It was created in 2015/2016 as a way of managing all Mono's products and subscriptions associated with the products. It's undergone many iterations since but it's core functionality has changed little.
```
Comment (siozosdr) [inline presentations/03_HAL/README.md]: its core functionality

**Pull request merged**